build.gradle

```diff
@@ -116,22 +110,19 @@ dependencies {
 
     implementation group: 'org.springframework.cloud', name: 'spring-cloud-starter-bootstrap', version: versions.springcloud
 
-    implementation group: 'org.springframework.cloud', name: 'spring-cloud-openfeign', version: versions.springcloud, {
-        exclude group: 'org.springframework.boot', module: 'spring-boot-starter-actuator'
-    }
+    implementation group: 'org.springframework.cloud', name: 'spring-cloud-openfeign', version: versions.springcloud
 
     implementation group:'com.fasterxml.jackson.datatype', name: 'jackson-datatype-json-org', version: versions.fasterXmlJackson
     implementation group:'com.fasterxml.jackson.datatype', name: 'jackson-datatype-hppc', version: versions.fasterXmlJackson
     implementation group:'com.fasterxml.jackson.datatype', name: 'jackson-datatype-jsr310', version: versions.fasterXmlJackson
-    implementation group:'com.fasterxml.jackson.datatype', name: 'jackson-datatype-hibernate5', version: versions.fasterXmlJackson
+    implementation group:'com.fasterxml.jackson.datatype', name: 'jackson-datatype-hibernate5-jakarta', version: versions.fasterXmlJackson
     implementation group:'com.fasterxml.jackson.core', name: 'jackson-annotations', version: versions.fasterXmlJackson
     implementation group:'com.fasterxml.jackson.core', name: 'jackson-databind',version: versions.fasterXmlJackson
     implementation group:'com.fasterxml.jackson.core', name: 'jackson-core', version: versions.fasterXmlJackson
     implementation group:'com.fasterxml.jackson.module', name: 'jackson-module-afterburner', version: versions.fasterXmlJackson
 
-    implementation group: 'javax.el', name: 'javax.el-api', version: '3.0.0'
-    implementation group: 'org.zalando', name: 'problem-spring-web', version: versions.zalando
-    implementation group: 'org.zalando', name: 'jackson-datatype-problem', version: versions.zalando
+    implementation group: 'org.zalando', name: 'problem-spring-web', version: '0.29.1'
+    implementation group: 'org.zalando', name: 'jackson-datatype-problem', version: '0.27.1'
     implementation group: 'org.springdoc', name: 'springdoc-openapi-ui', version: '1.7.0'
```

Comment:
`implementation group: 'org.springdoc', name: 'springdoc-openapi-starter-webmvc-ui', version: '2.1.0'`

you need to change'springdoc' to swagger work with spring3

Reply:
Okay I will do the change now